Math Magicians: refactor with hooks

djo1975 opened the pull request:
-Use React hooks.

Description
In this project, I refactored the Calculator component developed in the past projects. Instead of using class based components, I used hooks now.

djo1975 committed:
Add component with hooks

djo1975 committed:
Add project description and live link

djo1975 committed:
Add test section and react

DuaneDave reviewed (approved):
Hi @djo1975 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ App is working after refactoring
✔️ Hooks correctly used
✔️ Correct usage of gitflow

## Optional suggestions
- It would give a user better feedback of their actions when a button is pressed if you can add an active state to the buttons.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

djo1975 committed:
Add an active state to the buttons

djo1975 commented:
Hi @DuaneDave, thanks for suggestion and if it is optional I do next:
I've added a activeBtn state to track which button is currently active. I've also added a className prop to the button elements, which will apply the active class to the currently active button. Finally, I've updated the handleBtnClick function to set the activeBtn state to the name of the clicked button. Of course, I checked functionality and every thing is working well as well. :)
Thx a lot
